frontend/src/renderer/utils/api.tsx

```diff
@@ -14,7 +14,7 @@ const selectBaseUrl = () => {
   }
 };
 
-export const apiCall = async (endpoint, method, data) => {
+export const apiCall = async (endpoint, method, data, disableNotification = false) => {
```

Comment:
the default parameter should either be `true` or it should be `if(disabledNotification)`, otherwise we won't have any Notifcation.